src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -682,6 +682,25 @@ private static function get_wp_media(): array {
 			);
 		}
 
+		// Get the image format transforms.
+		$mappings           = wp_get_image_editor_output_format( '', '' );
+		$formatted_mappings = array();
+
+		if ( ! empty( $mappings ) ) {
+			foreach ( $mappings as $format => $mime_type ) {
+				$formatted_mappings[] = sprintf( '%s &rarr; %s', $format, $mime_type );
+			}
+			$mappings_display = implode( ', ', $formatted_mappings );
+		} else {
+			$mappings_display = __( 'No format transforms defined' );
+		}
+
+		$fields['image_format_transforms'] = array(
+			'label' => __( 'Image Format Transforms' ),
```

Comment:
match case of other entries
```suggestion
			'label' => __( 'Image format transforms' ),
```